index.js

```diff
@@ -21,8 +23,12 @@ function accesorString(value) {
 
 module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
+	// Change the request from an /abolute/path.js to a relative ./path.js
+	// This prevents [chunkhash] values from changing when running webpack
+	// builds in different directories.
+	var newRequestPath = "." + path.sep + path.basename(remainingRequest);
```

Comment:
This needs to be `path.relative` to the current module's context. And either of these will break if `remainingRequest` is not a path (it could be a loader expression, for example).